rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_storage.hpp

```diff
@@ -90,6 +93,7 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
   std::unordered_map<std::string, int> topics_;
   std::vector<rosbag2_storage::TopicMetadata> all_topics_and_types_;
   std::string relative_path_;
+  rosbag2_storage::StorageFilter storage_filter_ {};
```

Comment:
nit: Remove space before braces.

Reply:
I copied the format a few lines above (lines 90-92). Should I remove the space for all of them?

Reply:
I didn't see that it's done as such in the lines above. Nevermind then.